app/routes/pages/components/LandingPage.tsx

```diff
@@ -78,7 +78,15 @@ const LandingPage: PageRenderer = ({ loggedIn }) => {
               på vårt <Link to="/contact">kontaktskjema</Link> eller på
               e-postene under! På kontaktskjemaet har du mulighet til å sende
               anonyme beskjeder, og du kan lett velge hvem som skal få mailen
-              din.
+              din. Abakus har også en{' '}
+              <a
+                href="https://avvik.abakus.no"
+                rel="noopener noreferrer"
+                target="_blank"
+              >
+                varslingsportal
+              </a>{' '}
+              som vi anbefaler å bruke til anonym varsling.
```

Comment:
```suggestion
              som kan brukes til varsling.
```

Or something similar. It's not "anbefalt", it _should_ be used. and its both anonymous and not anonymous.

Reply:
To me, "anbefalt" means the same, but I'll change it 👍🏼

Reply:
```suggestion
              som kan brukes til anonym varsling.
```

I'd like to keep the anonymity part since it's obvious that a "varslingsportal" can be used for "varsling".